pkg/ocm/versions.go

```diff
@@ -45,7 +45,7 @@ func (c *Client) GetVersions(channelGroup string) (versions []*cmv1.Version, err
 		var response *cmv1.VersionsListResponse
 		response, err = collection.List().
 			Search(filter).
-			Order("default desc, id desc").
+			Order("id desc").
```

Comment:
If this is intended I'm okay to merge, @oriAdler could you confirm?

Reply:
The references for this function are:
1. Create cluster - getVersionList - Ok
2. Create service - getVersionList @renan-campos can you approve the change?
3. List versions - Ok
4. Create account roles and upgrade account roles - Ok
5. Create and upgrade operator roles - I think it should get the same version as creating account roles, as they go in combination.

I think we are safe to merge this change

Reply:
just to make sure, iiuc, that means in effect we no longer use the default version anywhere. New clusters will default to the latest version, when using interactive mode the first version suggested will be the newest one etc. In other words - the default version everywhere will always be the newest version.

but actually, looking at the code below, this sorting is ignored and the versions get re-sorted with the default at the top - so this change actually has no affect. (also, the id sort in the query doesn't make any sense anyways, since it will sort the versions by dictionary order and not by actual version comparison, which is also handled below)

Reply:
Good catch!
Missed that in the rush, thanks.